editor.planx.uk/src/pages/SubmissionDownload/VerifySubmissionEmail.tsx

```diff
@@ -0,0 +1,121 @@
+
+import { downloadZipFile } from "./helpers/downloadZip";
+import { VerifySubmissionEmailProps } from "./types";
+
+export const DOWNLOAD_APPLICATION_FILE_URL = `${
+  import.meta.env.VITE_APP_API_URL
+}/download-application-files`;
+
+const verifySubmissionEmailSchema = object({
+  email: string().email("Invalid email").required("Email address required"),
+});
+export const VerifySubmissionEmail = ({
+  params,
+}: VerifySubmissionEmailProps): JSX.Element => {
+  const { sessionId, team } = params;
+  const [downloadApplicationError, setDownloadApplicationError] = useState("");
+  const [loading, setLoading] = useState(false);
+
+  const formik = useFormik({
+    initialValues: {
+      email: "",
+    },
+    onSubmit: async (values, { resetForm }) => {
+      setDownloadApplicationError("");
+      setLoading(true);
+      const url = `${DOWNLOAD_APPLICATION_FILE_URL}/${sessionId}/?email=${encodeURIComponent(
+        values.email,
+      )}&localAuthority=${team}`;
+      try {
+        const { data } = await axios.get(url, {
+          responseType: "arraybuffer",
+        });
+        downloadZipFile(data);
+        resetForm();
+        setLoading(false);
+      } catch (error) {
+        setLoading(false);
+        if (isAxiosError(error)) {
+          setDownloadApplicationError(
+            "Sorry, something went wrong. Please try again.",
+          );
+          resetForm();
+        }
+        console.error(error);
+      }
+    },
+    validateOnChange: false,
+    validateOnBlur: false,
+    validationSchema: verifySubmissionEmailSchema,
+  });
+  return (
+    <Container maxWidth="contentWrap">
+      <Typography maxWidth="formWrap" variant="h1" pt={5} gutterBottom>
+        Download application
+      </Typography>
+      {loading ? (
+        <DelayedLoadingIndicator />
+      ) : (
+        <Box width="100%">
+          <Card handleSubmit={formik.handleSubmit}>
+            <ErrorWrapper error={downloadApplicationError}>
+              <>
+                <CardHeader
+                  title="Verify your submission email address"
+                  description="We will use this to confirm that you have access to the submission email inbox that is set up for your team. Entering the correct email address will start the file download automatically."
+                />
+                <InputRow>
+                  <InputLabel label={"Email address"} htmlFor={"email"}>
+                    <Input
+                      bordered
+                      errorMessage={
+                        formik.touched.email && formik.errors.email
+                          ? formik.errors.email
+                          : undefined
+                      }
+                      id="email"
+                      name="email"
+                      onBlur={formik.handleBlur}
+                      onChange={formik.handleChange}
+                      type="email"
+                      value={formik.values.email}
```

Comment:
nit: I noticed that this input is _not_ triggering an error when I press "enter" from within it before typing anything or when I just type "jess" (invalid email format) and press enter - which does not match our typical public-facing email input error handling !

Reply:
Ah wait I see what you mean now, pressing enter on the keyboard does not submit it (only pressing the continue button does), therefore the error handling is not triggered. Looking for a fix now...

Reply:
@jessicamcinchak just looking at other inputs in the editor and it seems to be normal behaviour that just pressing 'enter' does not do anything. If you press tab it correctly finds the 'continue' button that you can then press enter on. Am I missing something else?

Reply:
Sorry the enter key part was my mistake - only the "Postcode" input on FindProperty does the on-enter-key validation (special a11y case because the next address input is conditional, not visible yet, I believe), was mis-remembering that we do that throughout ! This is working for me now :+1:

Reply:
I do think now we have the summary table creating more distance between the 'Continue' button and the input, it makes it look a bit more confusing - it's not so clear that the continue button is for the input.

Reply:
Agree - maybe swap summary table to top & email input below? A nice specific design prompt for larger group maybe?